tests/foreman/cli/test_contentview.py

```diff
@@ -821,6 +913,49 @@ def test_positive_remove_version_by_id_from_composite(self):
 
     @tier2
     @run_only_on('sat')
+    def test_positive_remove_component_by_name(self):
+        """Create a composite content view and remove component from it by name
+
+        :id: 908f9cad-b985-4bae-96c0-037ea1d395a6
+
+        :expectedresults: Composite content view info output does not contain
+            any values for its components
+
+        :BZ: 1416857
+
+        :CaseLevel: Integration
+        """
+        # Create new repository
+        new_repo = make_repository({u'product-id': self.product['id']})
+        # Sync REPO
+        Repository.synchronize({'id': new_repo['id']})
+        # Create new content-view and add repository to view
+        new_cv = make_content_view({u'organization-id': self.org['id']})
+        ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'organization-id': self.org['id'],
+            u'repository-id': new_repo['id'],
+        })
+        # Publish a new version of CV
+        ContentView.publish({u'id': new_cv['id']})
+        # Get the CV info
+        new_cv = ContentView.info({u'id': new_cv['id']})
+        # Create a composite CV
+        comp_cv = make_content_view({
+            'composite': True,
+            'organization-id': self.org['id'],
+            'component-ids': new_cv['versions'][0]['id']
+        })
+        self.assertEqual(len(comp_cv['components']), 1)
+        ContentView.remove_version({
+            u'content-view-version-id': new_cv['versions'][0]['id'],
+            u'id': comp_cv['id'],
```

Comment:
Neither CV name, nor component name are specified here. Please update the test accordingly.